lib/index.ts

```diff
@@ -71,19 +79,55 @@ export const storyblokInit = (pluginOptions: SbSDKOptions = {}) => {
     loadBridge(bridgeLatest);
   }
 
+  // Rich Text resolver
+  richTextResolver = new RichTextResolver(richText.schema);
```

Comment:
does this support the scenario where richText.schema is not defined? 
richText's default value is `{}`, what happens if the user doesn't define richText configuration in `storyblokInit`?

Reply:
Yes, that case is already handled in storyblok-js-client. Basically, if no schema is passed then js-client uses the default one.